Automattic/wp-calypso
# Editing Toolkit: refactor and add Tracks recording

## Changes proposed in this Pull Request

To simplify this plugin I refactored from a switch to a lookup map and cleaned up the component that adds the link. This helps to prevent breakage when the description already contains links and other components. I also incorporated a tracks event record to help with stats.

## Testing instructions

1. Pull and go to `apps/editing-toolkit`
2. Run `yarn dev --sync`
3. Pull up your sandbox site and go to your post editor.
4. Check that the description is loading properly in the sidebar and the block picker.
5. This is my post that already has every block added for help 😃 sdffaJ-testing-p2
